lmod_proxy/edx_grades/actions.py

```diff
@@ -30,6 +30,16 @@ def post_grades(gradebook, form):
     csv_file.seek(0)
     results = None
     try:
+        # Create assignment to explicitly set max points.
+        assignment, max_points = _get_assignment_name_max_points(form)
+        gradebook.create_assignment(
+            assignment[0:3] + assignment[-2:],
+            assignment,
+            1.0,
+            max_points,
+            '12-15-2013',
+            gradebook.gradebook_id
+        )
         results, time_taken = gradebook.spreadsheet2gradebook(
```

Comment:
It looks like `csv_file` will need to be rewinded again, since `_get_assignment_name_max_points` read through it all